package.json

```diff
@@ -14,6 +14,7 @@
     "audit:resolve": "SHELL=sh resolve-audit --production",
     "audit:check": "SHELL=sh check-audit --production",
     "build": "tsc -p ./tsconfig.build.json",
+    "build:openapi": "multi-file-swagger ./src/interface/template.yaml > src/interface/swagger.json",
```

Comment:
Hmm we should probably also update the readme to recommend this - I'm sure there's docs we can borrow from `thirdparty-api-adapter` or `thirdparty-scheme-adapter`

Reply:
can't seem to find docs like that there but will update the readme